drivers/xen/gnttab.c

```diff
@@ -80,19 +78,6 @@ grant_ref_t gnttab_grant_access(domid_t domid, unsigned long frame, int readonly
 	return ref;
 }
 
-grant_ref_t gnttab_grant_transfer(domid_t domid, unsigned long pfn)
-{
-	grant_ref_t ref;
-
-	ref = get_free_entry();
-	gnttab_table[ref].frame = pfn;
-	gnttab_table[ref].domid = domid;
-	wmb();
-	gnttab_table[ref].flags = GTF_accept_transfer;
-
-	return ref;
-}
-
 int gnttab_end_access(grant_ref_t ref)
```

Comment:
I think we need to put comments as requested on the other patch: "Document functions, please."